Sources/SwiftInspectorVisitors/ImportVisitor.swift

```diff
@@ -34,6 +34,26 @@ public final class ImportSyntaxReader: SyntaxVisitor {
     return .visitChildren
   }
 
+  public override func visit(_ node: ProtocolDeclSyntax) -> SyntaxVisitorContinueKind {
```

Comment:
In an effort to make walking the tree with the import visitor more performant, I had it skip a bunch of declarations. There's likely a bunch more we could ignore, but I'm not sure how deep to go here.